packages/evolution-frontend/src/components/inputs/maps/google/InputMapGoogle.tsx

```diff
@@ -135,10 +136,13 @@ const InputMapGoogle: React.FunctionComponent<InputGoogleMapPointProps> = (props
     React.useEffect(() => {
         if (map && props.markers.length >= 1) {
             const mapBounds = new google.maps.LatLngBounds();
-            props.markers.forEach((marker) =>
+            const points = props.markers.map((marker) => marker.position.geometry) as [GeoJSON.Point];
```

Comment:
I think the syntax here would be `GeoJSON.Point[]` for an array of points. `[GeoJSON.Point]` is an array with a single point.